lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel

```diff
@@ -77,6 +77,18 @@ proc createRowLayoutforMayaReference(string $parent, string $rowLayoutName)
     }
 }
 
+proc string defaultMayaReferencePrimName() {
+    return python("import mayaUsdAddMayaReference; mayaUsdAddMayaReference.defaultMayaReferencePrimName()");
+}
+
+proc string defaultVariantSetName() {
+    return python("import mayaUsdAddMayaReference; mayaUsdAddMayaReference.defaultVariantSetName()");
+}
+
+proc string defaultVariantName() {
+    return python("import mayaUsdAddMayaReference; mayaUsdAddMayaReference.defaultVariantName()");
+}
```

Comment:
I had originally put the strings in the localization file, but after looking at the Pixar Usd code for MakeValidIdentifier I see they only accept English letters (and numbers). So I added the strings as define in the python file and added these MEL helpers to get them.